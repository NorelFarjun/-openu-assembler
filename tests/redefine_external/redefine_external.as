MAIN: add r3, r4
.external MAIN
