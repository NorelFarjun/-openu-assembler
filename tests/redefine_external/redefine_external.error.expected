1: incorrect definition 'external'
Bad input file - not outputting
